g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/conf/WebConfiguration.java

```diff
@@ -77,8 +77,10 @@
 
 	@Value("${vipservice.cross.domain.maxage}")
 	private String maxAge;
-	
-	
+
+	@Value("${translation.bundle.file.basepath}")
```

Comment:
Should this parameter be added in application.properties? I don't see any change for application.properties.

Reply:
This parameter exists in application.properties already, here is just to read it.